README.md

```diff
@@ -26,21 +33,62 @@ Package distribution: [pypi.org/project/dxfeed](https://pypi.org/project/dxfeed/
 
 **Requirements:** python >= 3.6
```

Comment:
I think not. Because at the moment, those versions of the library that have already been built, work under python 3.6. And users who import a project using pip will have no problems. 

And if we release a new version, then yes, we will either have to fix the build for 3.6, or remove it from the project.
My point is that this PR will not bump the version of the library, but just fix the CI/CD and update the README.md.